### PR TITLE
Add Associations and Better formatting to hover of model

### PR DESCRIPTION
**Pull request by rickcaudill**
Three small patches:

1. I wanted the ability to see the comment of the column.
2. I added the ability to see the associations to the hover of a model.  
3. I have also added better formatting to the table.

Screenshot 2024-01-27 at 12 18 17 PM

### CODE REVIEW

**Comment by andyw8** [comment]
👋 @rickcaudill thank you for the proposed contributions! We've discussed it within the team and have some thoughts:

- Let's split this up into three separate PRs, corresponding the features you list, to allow for easier review and discussion.
- It doesn't seem advantagous to use an ERB template – we could build-up the markdown in plain Ruby.
- We aren't sure that column comments are commonly used, but it could be useful to add other information to the hover, such as default values, or whether a column is nullable. What do you think?

File: Gemfile.lock [hunk 1]
```diff
@@ -289,7 +289,7 @@ DEPENDENCIES
   ruby-lsp-rails!
   sorbet-static-and-runtime
   sqlite3
-  tapioca (~> 0.12)
+  tapioca (~> 0.11)
```
[the remark this answers is not in the document]
Comment (rickcaudill) [reply Gemfile.lock:292]: Will do.

File: test/dummy/config/routes.rb [hunk 2]
```diff
@@ -2,4 +2,5 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  resources :users
```
[the remark this answers is not in the document]
Comment (rickcaudill) [reply test/dummy/config/routes.rb:5]: No, this was a remanent of another change for routes that I was working on..  Can back this out

**Comment by rickcaudill** [comment]
> 👋 @rickcaudill thank you for the proposed contributions! We've discussed it within the team and have some thoughts:
> 
> * Let's split this up into three separate PRs, corresponding the features you list, to allow for easier review and discussion.

Sure.  Will do this later today or tomorrow.

> * It doesn't seem advantagous to use an ERB template – we could build-up the markdown in plain Ruby.

The reason I used ERB here was inlining the markdown was failing to get the desired results I was looking for.  

> * We aren't sure that column comments are commonly used, but it could be useful to add other information to the hover, such as default values, or whether a column is nullable. What do you think?

I was definitely thinking about adding more columns.  The default is something I should have added for sure.  I think column comments matter in bigger databases. I think column comments are something DB Admins might like.  But they probably wouldn't be using an IDE.

**Comment by rickcaudill** [comment]
I will make these into smaller patches with the new resolver code.  I also have some more patches for viewing view information, as well as some other patches I am in the works of.

**Comment by rickcaudill** [comment]
Closing pull request to change direction based on comments.

**Pull request closed without merging**